Fix linking with --as-needed -- "undefined reference to `g_thread_init'"

GNOME Bug 595767

---
--- tool/gkr-tool.c
+++ tool/gkr-tool.c
@@ -25,6 +25,7 @@
 
 #include "gkr-tool.h"
 
+#include <glib.h>
 #include <glib/gi18n.h>
 #include <gtk/gtk.h>
 
--- tool/Makefile.am
+++ tool/Makefile.am
@@ -20,6 +20,7 @@
 	$(top_builddir)/gp11/libgp11.la \
 	$(top_builddir)/gcr/libgcr.la \
 	$(top_builddir)/egg/libegg.la \
+	$(GTHREAD_LIBS) \
 	$(GTK_LIBS) \
 	$(GCRYPT_LIBS) \
 	$(LIBTASN1_LIBS) \
